sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/metadata.yaml

```diff
@@ -6,6 +6,7 @@ owners:
   - kignasiak@mozilla.com
 scheduling:
   dag_name: bqetl_monitoring_airflow
+  date_partition_parameter: null
```

Comment:
So this is used by the DAG generation. It appears the default behavior (if this param is not set to null) is to set it to `submission_date`. This is why I'm forcing it to be `null` here.

As it happened here:
https://github.com/mozilla/bigquery-etl/pull/3278/files#diff-2870d4ae7225de06e4652fcaed3621c0be0db0f5f03db4fffdd264d71aa5287fL70

Reply:
Yeah it is set to submission_date by default. I was wondering because in the DAG  bqetl_analytics_aggregations is set to None without forcing the Null in the metadata. I did a search in github and it was changed in this [PR to introduce date_partition_offset](https://github.com/mozilla/bigquery-etl/commit/c9520d6112c6d65f32025c657f989efbddaa3e9c#diff-4cb089996f5af781b0342647073e4b030aa633bcc5b93ab7b4fc9f1d97dd5f87).